public-ui/src/app/data-browser/chart/chart.component.ts

```diff
@@ -47,208 +13,201 @@ const DATA_LABEL_STYLE = {
+    }*/
 
-              enabled: false
-          },
-          title: options.title,
-          subtitle: {
-          },
-          tooltip: {
-            pointFormat: '<b>{point.y} </b><br>{series.name}'
-          },
-          plotOptions: {
-              series: {
-                  animation: {
-                      duration: 100,
-                  },
-                  pointWidth: options.pointWidth ? options.pointWidth : null,
-                  minPointLength: 3,
-                  events: {
-                    click: function (event) {
-                      console.log('plot options clicked ',  event.point);
-                    }
-                  }
-              },
-              pie: {
-                  borderColor: null,
-                  slicedOffset: 4,
-                  dataLabels: {
-                      enabled: true,
-                      style: DATA_LABEL_STYLE,
-                      distance: -30,
-                      format: '{point.name} {point.percentage:.0f}%'
-                  }
-              },
-              column: {
-                  shadow: false,
-                  borderColor: null,
-                  colorByPoint: true,
-                  groupPadding: 0,
-                  dataLabels: {
-                      enabled: false,
-                  },
-                events: {
-
-                }
-              },
-              bar: {
-                shadow: false,
-                borderColor: null,
-                colorByPoint: true,
-                groupPadding: 0,
-                dataLabels: {
-                  enabled: false,
-                },
-                events: {
-                }
-              }
-          },
-          yAxis: {
-            title: {
-              text: null
-            },
-            lineWidth: 1,
-            lineColor: '#979797',
-            gridLineColor: this.backgroundColor
-          },
-          xAxis: {
-            categories: options.categories,
-            type: 'category',
-            labels: {
-                style: {
-
-                    whiteSpace: 'nowrap',
-                }
-            },
-            lineWidth: 1,
-            lineColor: '#979797'
+    return {
+      chart: options.chart,
+      lang: {
+        noData: {
+        style: {
+          fontWeight: 'bold',
+          fontSize: '15px',
+          color: '#303030'
+        }
+        }
+      },
+      credits: {
+        enabled: false
+      },
+      title: options.title,
+      subtitle: {},
+      tooltip: {
+        pointFormat: '<b>{point.y} </b><br>{series.name}'
+      },
+      plotOptions: {
+        series: {
+          animation: {
+            duration: 100,
           },
-          zAxis: {
+          pointWidth: options.pointWidth ? options.pointWidth : null,
+          minPointLength: 3,
+          events: {
+            click: function (event) {
+              console.log('plot options clicked ', event.point);
```

Comment:
These kinds of debugging statements should probably not be checked in

Reply:
Yeah. I am just going to have to add them back when we get into drilling down by clicking plots. That is why i left them. More like a todo. We will want to capture analytics too when people click a plot.